package.json

```diff
@@ -1,13 +1,13 @@
 {
 	"name": "@fandom/jwplayer-fandom",
-	"version": "1.12.1",
+	"version": "2.0.0-beta-1",
 	"description": "JWPlayer for Fandom",
-	"main": "dist/bundle.umd.js",
-	"module": "dist/bundle.esm.js",
-	"types": "dist/main.d.ts",
+	"main": "bundle.umd.js",
+	"module": "bundle.esm.js",
+	"types": "main.d.ts",
 	"install-peers": "install-peers",
 	"scripts": {
-		"build": "yarn run create-dist && rollup -c",
+		"build": "yarn run create-dist && rollup -c && cp package.json dist/ && cd dist && npm set-script prepare ''",
```

Comment:
Also, what is the '' at the end of the `npm set-script prepare ''` portion in the build command?

Reply:
This removes the `husky` command which doesn't work in the dist directory because it's not the root .git directory